### PR TITLE
Generate CRD example

### PR DESCRIPTION
**Pull request by kakkoyun**
It may not be the optimum way to generate but better than nothing I believe.

cc @nmagnezi @rollandf

### CODE REVIEW

File: example/main.jsonnet [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        hashrings:: null,
+      },
+      apiGateway: {
+        image: obs.config.apiGateway.image,
+        version: obs.config.apiGateway.version,
+      },
+    },
+  },
+
+  manifests+:: {
```
Comment (brancz) [inline example/main.jsonnet]: I don’t think we need to apply this pattern for this, just declare the crd directly I would say.

File: example/main.jsonnet [hunk 2]
```diff
@@ -0,0 +1,52 @@
+local obs = (import '../environments/base/observatorium.jsonnet');
+
+{
+  'observatorium-crd': {
```
Comment (brancz) [inline example/main.jsonnet]: I don’t think this is even necessary just wrote the cr directly

Comment (kakkoyun) [reply example/main.jsonnet]: I hope this time, I understand you correctly :)

File: build_dev.sh [hunk 3]
```diff
@@ -14,5 +14,14 @@ mkdir environments/dev/manifests
...
+rm -rf tests/manifests
+mkdir tests/manifests
+
 jsonnet -J vendor -m tests/manifests tests/main.jsonnet | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml' -- {}
 find tests/manifests -type f ! -name '*.yaml' -delete
+
+rm -rf example/manifests
+mkdir example/manifests
+
+jsonnet -J vendor example/main.jsonnet | gojsontoyaml >example/manifests/observatorium-crd.yaml
```
Comment (brancz) [inline build_dev.sh]: I think this should be in a follow up PR, as I don't want to change more than necessary in this PR, but `-crd.yaml` is misleading, as this is a custom resource instance not a custom resource definition. I think in a follow up we should just rename this to `observatorium.yaml`.

**Comment by brancz** [comment]
Not sure why but there still seems to be a CI failure.

**Comment by kakkoyun** [comment]
CI is a bit flaky. I don't know the exact reason. I'm on a bad strake with CI runs recently, CIs just hate me.
I'll re-run it if it fails.

**Pull request merged**